packages/flutter/test/widgets/gesture_detector_test.dart

```diff
@@ -367,9 +367,21 @@ void main() {
         ),
       ),
     );
-
-    final TestGesture gesture = await tester.startGesture(const Offset(400.0, 50.0));
     const int pointerValue = 1;
+    const Offset offset = Offset(400, 50);
```

Comment:
Perhaps just make a const on top of this file for these initial down events in this file and in the next one.